01_P/P_2_2_4_01/sketch.js

```diff
@@ -31,14 +32,12 @@ var r = []; // radius
 
 function setup() {
   createCanvas(600,600);
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).